README.md

```diff
@@ -1,11 +1,62 @@
 # README
 
-Add your handler readme here. Remember to include the following:
+## What is this?
 
-- Tell people how to install it (e.g. pip install ...).
-- Be clear about the purpose of the template handler, its capabilities and limitations.
-- Tell people how to use it.
-- Give examples of the template handler in use.
+`sceptre-cdk-handler` is a TemplateHandler for Sceptre (versions 2.7 and up) that lets you use an
+AWS CDK template as a stack's template.
 
-If you have any questions or encounter an issue
-[please open an issue](https://github.com/Sceptre/project/issues/new)
+This template handler will use the AWS CDK to synthesize the CDK stack into a CloudFormation template
+and then run `npx cdk-assets` to publish any required assets to S3.
+
+**By using the CDK Handler, you are letting CDK synthesize a template, and upload artifacts to S3,
```

Comment:
CDK isn't just uploading artifacts to S3. Depending on how your have your CDK configure, it might be pushing images to ECR, or doing other sorts of "pre-deploy" actions. I suspect you'll need to change this line to be more specific to what the CDK handler is doing.